timescaledb/overview/release-notes/index.md

```diff
@@ -82,6 +85,40 @@ for more information and links to installation instructions when upgrading from
 
 ## Release notes
 
+## 2.6.0 (2022-02-16)
+This release adds major new features since the 2.5.2 release, among
+them:
+
+* Compression in continuous aggregates
+* Experimental support for timezones in continuous aggregates
+* Experimental support for monthly buckets in continuous aggregates
```

Comment:
same comments as above, we also support years

Reply:
Do you need to mention "years" here as well?